sdk/include/reactos/undocuser.h

```diff
@@ -255,9 +255,7 @@ C_ASSERT(sizeof(MSGBOXDATA) == 0x88);
 
 #else
 
-#if (_WIN32_WINNT <= _WIN32_WINNT_WIN2K)    /* (NTDDI_VERSION <= NTDDI_WIN2KSP4) */
-C_ASSERT(sizeof(MSGBOXDATA) == 0x48);
-#elif (_WIN32_WINNT >= _WIN32_WINNT_WIN7)   /* (NTDDI_VERSION >= NTDDI_WIN7) */
+#if (_WIN32_WINNT >= _WIN32_WINNT_WIN7) /* (NTDDI_VERSION >= NTDDI_WIN7) */
```

Comment:
Please revert.

Reply:
Reverted, though I wonder why this code is still wanted. (Both files.)

Reply:
The person(s) who wrote that code, want to document what the size of this undocumented structure is on these different Windows versions.

Reply:
It seems you were the author, 3 years ago. Even at that time, ReactOS (build) did not care about 2000 anymore, did it?
If not removing this "documentation", I would suggest to switch to some kind of comment. Anyway...